src/App.jsx

```diff
@@ -28,6 +28,14 @@ function App() {
       <p className="read-the-docs">
         Click on the Vite and React logos to learn more
       </p>
```

Comment:
You are going to want to remove the residual code from the Vite setup.  You can get rid of this whole <p>